### ISSUE TITLE
Update old Spring MVC design to fsa-style latest

### ISSUE THREAD

**Issue by francisrupert** [USDA-FSA/ref-app-settings#2]
### Background

The [previous Spring MVC prototype](https://sharepoint.fsa.usda.net/mgr/itsd/architecture/EA_Team/Shared%20Documents/ARCHITECTURE%20DOMAINS/XCut%20-%20UX/8.%20Projects/Reference%20Apps/Spring%20MVC/04%20design/prototype/spring-mvc-app--initial/_templates.html) was delivered by @francisrupert in Summer of 2016 as an early round exploration of the Design System. Now that the [Design System](http://usda-fsa.github.io/fsa-design-system/), and [fsa-style](http://usda-fsa.github.io/fsa-style/) CSS Framework has gained maturity/stability, this project updates its Design -- with the goal of the AO office dev team integrating it into their Spring MVC training materials.

This intentionally precedes design and prototyping work for the [SPA-based Inspections Reference App](https://github.com/USDA-FSA/ref-app-inspections/) as much of it can and will be folded into that app. Just as much of this can and will be folded into the Design System itself as static examples.

### General Tasking

- [x] Update Design, heavily inspired by PCP's UX Prototype.
- [x] Propose features to add that weren't part of previous version of Spring MVC prototype/ref-app
- [x] Keep in mind how this will be reused by [SPA-based Inspections Reference App](https://github.com/USDA-FSA/ref-app-inspections/)
- [x] Peer Reviewed

**Comment by francisrupert** [comment]
Adi (@arosenblum), Aaron (@atdiff), and Casey (@bignamehere),

### Action Requested

I have redesigned the Spring MVC prototype, viewable at the below Sharepoint links. Feedback is appreciated as comments on [USDA-FSA/ref-app-settings/issues/2](https://github.com/USDA-FSA/ref-app-settings/issues/2), though of course let me know if we need a formal meeting to review the below -- let alone discuss how/if this can update your Spring MVC training materials and documentation. 

### Details
 
* [Old](https://sharepoint.fsa.usda.net/mgr/itsd/architecture/EA_Team/Shared%20Documents/ARCHITECTURE%20DOMAINS/XCut%20-%20UX/8.%20Projects/Reference%20Apps/Spring%20MVC/04%20design/prototype/spring-mvc-app--initial/_templates.html)
* [Redesign](https://sharepoint.fsa.usda.net/mgr/itsd/architecture/EA_Team/Shared%20Documents/ARCHITECTURE%20DOMAINS/XCut%20-%20UX/8.%20Projects/Reference%20Apps/Spring%20MVC/04%20design/prototype/ref-app-settings/MASTER/index.html)

### Screencasts
* [Overview](https://sharepoint.fsa.usda.net/mgr/itsd/architecture/EA_Team/Shared%20Documents/ARCHITECTURE%20DOMAINS/XCut%20-%20UX/8.%20Projects/Reference%20Apps/Spring%20MVC/04%20design/screencasts/01__overview.webm)
* [Employees, Add](https://sharepoint.fsa.usda.net/mgr/itsd/architecture/EA_Team/Shared%20Documents/ARCHITECTURE%20DOMAINS/XCut%20-%20UX/8.%20Projects/Reference%20Apps/Spring%20MVC/04%20design/screencasts/02__employees-add.webm)
* [Employees, Edit](https://sharepoint.fsa.usda.net/mgr/itsd/architecture/EA_Team/Shared%20Documents/ARCHITECTURE%20DOMAINS/XCut%20-%20UX/8.%20Projects/Reference%20Apps/Spring%20MVC/04%20design/screencasts/03__employees-edit.webm)
* [Employee, Remove](https://sharepoint.fsa.usda.net/mgr/itsd/architecture/EA_Team/Shared%20Documents/ARCHITECTURE%20DOMAINS/XCut%20-%20UX/8.%20Projects/Reference%20Apps/Spring%20MVC/04%20design/screencasts/03__employees-edit--remove.webm)
* [Territories, Edit](https://sharepoint.fsa.usda.net/mgr/itsd/architecture/EA_Team/Shared%20Documents/ARCHITECTURE%20DOMAINS/XCut%20-%20UX/8.%20Projects/Reference%20Apps/Spring%20MVC/04%20design/screencasts/04__territories-edit.webm)

### Annotation Notes

In addition to the "hotspot hints" you've seen in play before, I've added very light annotation viewable via the same Hint toggle. They are best viewable while browser is full-screen.

**Comment by AdiRosenblum-USDA** [comment]
Thanks for this!  Will review asap!

**Comment by francisrupert** [comment]
@fraas 

asdfasdf

**Comment by bignamehere** [comment]
@francisrupert 

I reviewed both applications and the updates look good. However, it would be a good idea to have a review meeting to go over the two to gain a better understanding about what was accomplished and how the new features will be utilized. I did not have any exposure to the reference app until just recently, so a discussion might be nice.

We can also go over what is needed for next steps and align on any future development/tasks.
